### DISCUSSION TITLE
How to create separate notes for every annotation in a PDF (or multiple files) in one click?

### DISCUSSION THREAD

**Discussion by growingkittens** [windingwind/zotero-better-notes#888] [Q&A]
I am using **Zotero 6.0.30** with **Zotero Better Notes 1.0.4**.
My annotations are made on scanned PDFs with the "Select Area" tool.
This is a drawing of the workflow I am trying to achieve:
![signal-2024-02-09-062001_002](https://github.com/windingwind/zotero-better-notes/assets/143684113/ac20b52d-0d69-42dd-8934-06e904321aea)

**Comment by sjgknight** [comment]
I'm also trying to work this out. 
I have multiple annotations using the Zotero annotation tool (on snapshots and pdfs), and I want to create child notes from each annotation, applying the tag from the annotation to the childnote. 
I'm trying to work out if that's within scope for better-notes or needs to be some other javascript (because it's not in the core product and probably won't be)...
